src/pages/postgraphile/why-nullable.md

```diff
@@ -192,7 +192,10 @@ GraphQLNonNull-wrapped version of their original type:
 ```js
 module.exports = function NonNullRelationsPlugin(builder) {
   builder.hook("GraphQLObjectType:fields:field", (field, build, context) => {
-    if (!context.scope.isPgForwardRelationField) {
+    if (
+      !context.scope.isPgForwardRelationField ||
+      !context.scope.pgFieldIntrospection?.keyAttributes?.at(0)?.isNotNull
```

Comment:
```suggestion
      !context.scope.pgFieldIntrospection?.keyAttributes?.every(attr => attr.isNotNull)
```

Reply:
thanks for the suggestion, this looks more correct! tested the change and it works in my project